new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleValidation.java

```diff
@@ -0,0 +1,341 @@
+// Every test class needs to tagged with this annotation for log collection, diagnostic messages logging
+// and namespace creation.
+@IntegrationTest
+class ItSimpleValidation implements LoggedTest {
+
+  private static final String WDT_MODEL_FILE = "model1-wls.yaml";
+  private static final String IMAGE_NAME = "test-mii-image-2";
+  private static final String IMAGE_TAG = "v1";
+  private static final String APP_NAME = "sample-app";
+
+  private String opNamespace = null;
+  private String domainNamespace1 = null;
+
+  final String domainUid = "domain1";
+  String serviceAccountName;
+
+  /**
+   * Setup for test suite. Creates service account, namespace, and persistent volumes.
+   * @param namespaces injected by Junit extension
+   */
+  @BeforeAll
+  public void setup(@Namespaces(2) List<String> namespaces) {
+
+    // get a new unique namespace
+    logger.info("Assigning unique namespace for Operator");
+    assertNotNull(namespaces.get(0), "Namespace is null");
+    opNamespace = namespaces.get(0);
+
+    logger.info("Assigning unique namespace for Domain");
+    assertNotNull(namespaces.get(0), "Namespace is null");
+    domainNamespace1 = namespaces.get(1);
+
+
+    // Create a service account for the unique namespace
+    serviceAccountName = opNamespace + "-sa";
+    V1ServiceAccount serviceAccount = assertDoesNotThrow(
+        () -> Kubernetes.createServiceAccount(new V1ServiceAccount()
+            .metadata(new V1ObjectMeta().namespace(opNamespace).name(serviceAccountName))));
+    logger.info("Created service account: {0}", serviceAccount.getMetadata().getName());
+
+    // create persistent volume and persistent volume claim
+    String pvcName = domainUid + "-pvc"; // name of the persistent volume claim
+    String pvName = domainUid + "-pv"; // name of the persistent volume
+
+    V1PersistentVolumeClaim v1pvc = new V1PersistentVolumeClaim()
+        .spec(new V1PersistentVolumeClaimSpec()
+            .addAccessModesItem("ReadWriteMany")
+            .storageClassName(domainUid + "-weblogic-domain-storage-class")
+            .volumeName(pvName)
+            .resources(new V1ResourceRequirements()
+                .putRequestsItem("storage", Quantity.fromString("10Gi"))))
+        .metadata(new V1ObjectMetaBuilder()
+            .withName(pvcName)
+            .withNamespace(domainNamespace1)
+            .build()
+            .putLabelsItem("weblogic.resourceVersion", "domain-v2")
+            .putLabelsItem("weblogic.domainUid", domainUid));
+
+    boolean success = assertDoesNotThrow(
+        () -> TestActions.createPersistentVolumeClaim(v1pvc),
+        "Persistent volume claim creation failed, "
+        + "look at the above console log messages for failure reason in ApiException responsebody"
+    );
+    assertTrue(success, "PersistentVolumeClaim creation failed");
+
+    V1PersistentVolume v1pv = new V1PersistentVolume()
+        .spec(new V1PersistentVolumeSpec()
```

Comment:
this will need to be refactored out to a pv handler that takes the type as a parameter, we will need to support multiple types of storage providers